README.rst

```diff
@@ -106,11 +116,11 @@ test to run:
 Note that when teardown fails, two reports are generated for the case, one for
 the test case and the other for the teardown error.
 
-You can also specify the re-run delay time in the marker:
+You can also specify the re-run delay time in the marker and backoff:
```

Comment:
Consider reordering to `delay time and backoff in the marker`

Reply:
Done